core/assets/bundles/bundle.properties

```diff
@@ -520,6 +518,17 @@ rules.respawntime = Respawn Time:[LIGHT_GRAY] (sec)
 rules.wavespacing = Wave Spacing:[LIGHT_GRAY] (sec)
 rules.buildcostmultiplier = Build Cost Multiplier
 rules.buildspeedmultiplier = Build Speed Multiplier
+rules.waitForWaveToEnd = Begin counting to next wave after all enemies are killed
```

Comment:
```suggestion
Count Down to Next Wave
```

Reply:
I guess this one ain't accurate

Reply:
Count down to next wave as all enemies are destroyed

2 chars shorter. Maybe better

Reply:
So basically it does not count down after the wave has fired but instead after all enemies are killed, is what I am getting here.

`Start wave timer when all enemies are destroyed` fits better because `as` suggests `while` all the enemies are being destroyed.